RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/ProjectManager/activities/ActivityChapterList.java

```diff
@@ -272,4 +216,19 @@ private void prepareChapterCardData() {
         }
     }
 
+    @Override
+    public void onTaskComplete(int taskTag, int resultCode) {
+        if (resultCode == TaskFragment.STATUS_OK) {
+            if (taskTag == DATABASE_RESYNC_TASK) {
+                mDbResyncing = false;
+                refreshChapterCards();
+            } else if (taskTag == COMPILE_CHAPTER_TASK) {
+                ProjectDatabaseHelper db = new ProjectDatabaseHelper(ActivityChapterList.this);
+                for (int i : mChaptersCompiled) {
+                    mAdapter.notifyItemChanged(i);
+                    db.setCheckingLevel(mProject, i + 1, 0);
```

Comment:
If it doesn't affect anything, then this is fine. But I thought we'd want to set the checking level first before notifying the change.